doc/source/extending-jupyterhub.rst

```diff
@@ -176,3 +176,26 @@ report the current ``Ready/NotReady`` status of all nodes in the cluster.
      size for the workshop. This workflow also helps you avoid scrambling on
      the workshop day to set up the cluster and JupyterHub.
    - **After the workshop:** The cluster can be deleted.
+
+
+Setting up HTTPS
+----------------
+
+Zero to JupyterHub makes setting up HTTPS quite easy.
+
+1. Buy a domain name from a registrar. Pick whichever one you want
+2. Create an `A record` from the domain you want to use, pointing to the
```

Comment:
double backticks

Reply:
"point" instead of "pointing", and make it a new sentence.

Reply:
You can not create an A record without it pointing anywhere, so I'm going to leave this one as is for now unless you have strong opinions :)